test/e2e/network/conntrack.go

```diff
@@ -279,6 +279,158 @@ var _ = common.SIGDescribe("Conntrack", func() {
 		}
 	})
 
+	ginkgo.It("should be able to preserve TCP traffic when server pod cycles for a NodePort service", func() {
+
+		// Create a NodePort service
+		tcpJig := e2eservice.NewTestJig(cs, ns, serviceName)
+		ginkgo.By("creating a TCP service " + serviceName + " with type=NodePort in " + ns)
+		tcpService, err := tcpJig.CreateTCPService(func(svc *v1.Service) {
+			svc.Spec.Type = v1.ServiceTypeNodePort
+			svc.Spec.Ports = []v1.ServicePort{
+				{Port: 80, Name: "tcp", Protocol: v1.ProtocolTCP, TargetPort: intstr.FromInt(80)},
+			}
+		})
+		framework.ExpectNoError(err)
+
+		// Create a pod in one node to create the UDP traffic against the NodePort service every 5 seconds
+		ginkgo.By("creating a client pod for probing the service " + serviceName)
+		clientPod := e2epod.NewAgnhostPod(ns, podClient, nil, nil, nil)
+		nodeSelection := e2epod.NodeSelection{Name: clientNodeInfo.name}
+		e2epod.SetNodeSelection(&clientPod.Spec, nodeSelection)
+		cmd := fmt.Sprintf(`date; for i in $(seq 1 3000); do echo "$(date) Try: ${i}"; echo hostname | nc -w 5 -p %d %s %d; echo; done`, srcPort, serverNodeInfo.nodeIP, tcpService.Spec.Ports[0].NodePort)
```

Comment:
Even with netcat command , i am not able to see TCP SYN packet going. 
I tried this below option manually ,
nc -w 5 -p <src-port> <cluster-ip> <target-port>